src/app/kbv/controllers/load-question.test.js

```diff
@@ -14,6 +15,13 @@ describe("Load Question controller", () => {
     res = setup.res;
     next = setup.next;
 
+    req.session.tokenId = "dummy-session-id";
+
+    req.headers = {
+      "txma-audit-encoded": "dummy-txma-header",
```

Comment:
And same comment about test setup here

https://github.com/govuk-one-login/ipv-cri-address-front/pull/856/files#r1604492327

Reply:
updated